### ISSUE TITLE
Increase the size of the validation and test set, then retrain the 6 models. Currently, there are 7700 in the training set, 270 in the validation and 270 in the test set.

### ISSUE THREAD

**Issue by shuaigezhu** [shuaigezhu/star_galaxy_classification#38]

**Comment by shuaigezhu** [comment]
have increased the size of the validation set to 800, the test set to 800
have decreased the size of training set to 6500

lack of data is a future issue
